### DISCUSSION TITLE
Одобрить или добавить официальный свод законов Штата.

### DISCUSSION THREAD

**Discussion by HoRRiger** [octothorp-team/public-dobrograd#680] [Предложения]
Мое предложение состоит в том, чтобы одобрить или создать законы Штата. Это поможет, что полиции, что и игрокам в РП составляющей.

**Comment by Ggman77** [comment]
Я за и ещё раз ЗА, это добавит ролеплэй составляющей реализма, и законы не будут пустовать

**Comment by LENFYS** [comment]
За, могу даже поделиться если найду
